sdcm/nemesis.py

```diff
@@ -3665,7 +3682,16 @@ def wrapper(*args, **kwargs):  # pylint: disable=too-many-statements
                              node=args[0].target_node, publish_event=True) as nemesis_event:
             nemesis_info = argus_create_nemesis_info(nemesis=args[0], class_name=class_name,
                                                      method_name=method_name, start_time=start_time)
+            nemesis_run_info_key = f"{id(args[0])}--{method_name}"
             try:
+                NEMESIS_LOCK.acquire()  # pylint: disable=consider-using-with
+                if not is_exclusive:
+                    NEMESIS_RUN_INFO[nemesis_run_info_key] = "Running"
+                    NEMESIS_LOCK.release()
+                else:
+                    while NEMESIS_RUN_INFO:
```

Comment:
is it added for parallel nemesis?

Reply:
Yes, if you review per-commit then it will be much clearer. It is added as a separate commit.